custodian/policies/ec2/ec2-unmark-encrypted.yml

```diff
@@ -0,0 +1,37 @@
+policies:
+  - name: ec2-unmark-encrypted
+    mode:
+      type: periodic
+      schedule: "rate(1 day)"
+      role: arn:aws:iam::{account_id}:role/CustodianUnmarkEncryptedEC2
+    resource: aws.ec2
+    description: |
+      Unmarks any EC2 instances scheduled for deletion if they are now using encrypted EBS
+    filters:
+      - type: ebs
+        key: Encrypted
+        value: True
+      - "tag:maid_status": not-null
```

Comment:
Should that be "maid"?

Reply:
It's based on [this example](https://cloudcustodian.io/docs/aws/examples/ebsgarbagecollect.html)